fixing faq (#6226)
scaleway/docs-content: pages/iam/faq.mdx

## Quotas and limitations

### How many API keys can I create?
Each Organization can have up to 100 API keys by default. You can open a support ticket to ask for more quota.

### How many IAM applications can I create?

> Each Organization can have up to 100 IAM applications by default. You can open a support ticket to ask for more quota.

Reviewer: ```suggestion
Each Organization can have up to 100 IAM applications by default. You can submit a support ticket to request additional quota.
```

### How many IAM groups can I create?
Each Organization can have up to 50 IAM groups by default. You can open a support ticket to ask for more quota.

### How many IAM policies can I create?
Each Organization can have up to 50 IAM policies by default. You can open a support ticket to ask for more quota.

### How many users can my Organization have?

> Each Organization can have up to 50 users by default. You can open a support ticket to ask for more quota.

Reviewer: ```suggestion
Each Organization can have up to 50 users by default. You can submit a support ticket to request additional quota.
```
